tracer/dependabot/generated_calltargets.g.json

```diff
@@ -0,0 +1,14429 @@
+[
+  {
+    "IntegrationName": "AdoNet",
+    "AssemblyName": "netstandard",
+    "TargetTypeName": "System.Data.Common.DbCommand",
+    "TargetMethodName": "ExecuteDbDataReader",
+    "TargetReturnType": "System.Data.Common.DbDataReader",
+    "TargetParameterTypes": [
+      "System.Data.CommandBehavior"
+    ],
+    "MinimumVersion": {
+      "Item1": 2,
+      "Item2": 0,
+      "Item3": 0
```

Comment:
I know this is not intended to be read by a human, but `Item1` Item2` and `Item3`? :D

Reply:
Yep, the version struct is defined as a Tuple...